receiver/prometheusreceiver/README.md

```diff
@@ -128,6 +128,7 @@ receivers:
       trim_metric_suffixes: true
       use_start_time_metric: true
       start_time_metric_regex: foo_bar_.*
+      use_collector_start_time_fallback: true
```

Comment:
```suggestion
      use_collector_start_time_fallback: false
```

Should we give the default value as example? :P

Reply:
we use a non-default for all other options, so this seems ok to me